src/cmd/services/m3comparator/main/querier.go

```diff
@@ -129,8 +131,14 @@ func (q *querier) FetchCompressed(
 		}
 	}
 
+	const blockSize = time.Hour * 12
```

Comment:
Should this perhaps be a field on the `querier` itself? Seems like something we may want to change at some later point.

Reply:
Good point.